src/Apps/Sell/Components/SubmissionHeader.tsx

```diff
@@ -1,15 +1,19 @@
 import ArtsyLogoIcon from "@artsy/icons/ArtsyLogoIcon"
+import CloseIcon from "@artsy/icons/CloseIcon"
 import { Flex, FullBleed } from "@artsy/palette"
 import { AppContainer } from "Apps/Components/AppContainer"
 import { HorizontalPadding } from "Apps/Components/HorizontalPadding"
 import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
 import { Sticky } from "Components/Sticky"
 import { RouterLink } from "System/Components/RouterLink"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
+import { Media } from "Utils/Responsive"
 
 export const SubmissionHeader: React.FC = () => {
   const context = useSellFlowContext()
-
+  const isLastStep = context?.state?.isLastStep
   const submissionID = context?.state?.submissionID
+  const isSubmissionCompleted = isLastStep && submissionID
```

Comment:
In theory the last step should always have a submission ID, so this should t be needed. But I'll leave it up to you.

Reply:
Yes, but it was causing some issues on mWeb so I thought to add an extra check to be extra safe.

Reply:
Hmm, which issue does it cause? If it causes any issues, we should have a look to see if we can find out why and if we need to fix it.